db/migrate/20161221074640_link_documents_to_content_items.rb

```diff
@@ -0,0 +1,6 @@
+class LinkDocumentsToContentItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :document_id, :integer
+    add_foreign_key :content_items, :documents
```

Comment:
Do we need to add a non-null constraint migration to go out as part of the documents and editions branch?